application/src/main/java/org/opentripplanner/apis/transmodel/TransmodelGraphQLSchema.java

```diff
@@ -1300,40 +1302,19 @@ private GraphQLSchema create() {
               .build()
           )
           .dataFetcher(environment -> {
-            List<FeedScopedId> lineIds = mapIDsToDomainNullSafe(
-              environment.getArgumentOrDefault("lines", List.of())
-            );
-            List<String> privateCodes = environment.getArgumentOrDefault("privateCodes", List.of());
-            List<LocalDate> activeServiceDates = environment.getArgumentOrDefault(
-              "activeDates",
-              List.of()
-            );
+            var authorities = mapIDsToDomainNullSafe(environment.getArgument("authorities"));
+            var lineIds = mapIDsToDomainNullSafe(environment.getArgument("lines"));
+            var privateCodes = environment.<List<String>>getArgument("privateCodes");
+            var activeServiceDates = environment.<List<LocalDate>>getArgument("activeDates");
```

Comment:
Since I added the CriteriaCollection instead of using an Optional I'll leave this as it is.
<!-- Reviewable comment -OB_e0-x2RDhDV4mZeC9:bm9kn5w -->
<!-- Sent from Reviewable.io -->